doc/starter-pack.md

```diff
@@ -121,20 +121,20 @@ Clone the project with (standard, mobile or/and common) : please see all availab
 
 Example for Java plugin (with SSH) :
 ```sh
-git clone git@github.com:green-code-initiative/ecoCode-java.git
+git clone git@github.com:green-code-initiative/creedengo-java.git
 ```
 
-*WARNING* : if you are a new contributor (an not identified project `maintainer`), you have to use FORK / Pull Request System like explained here <https://github.com/green-code-initiative/ecoCode-common/blob/main/doc/HOWTO.md#howto-develop-in-open-source-mode>
+*WARNING* : if you are a new contributor (an not identified project `maintainer`), you have to use FORK / Pull Request System like explained here <https://github.com/green-code-initiative/creedengo-common/blob/main/doc/HOWTO.md#howto-develop-in-open-source-mode>
 
 ## Start local environment
 
 You will find all steps to start and configure your local Sonarqube dev Environment here :
 
-- 1st step - build your local plugin: <https://github.com/green-code-initiative/ecoCode-common/blob/main/doc/HOWTO.md#howto-build-the-sonarqube-ecocode-plugins>
-- 2nd step - launch local Sonarqube (with installation of previous local plugin built) : <https://github.com/green-code-initiative/ecoCode-common/blob/main/doc/HOWTO.md#howto-install-sonarqube-dev-environment>
-- 3rd step - check that local environment is running perfectly : choose one of repositories with suffix "test-project" (ex : <https://github.com/green-code-initiative/ecoCode-java-test-project/tree/main>)
+- 1st step - build your local plugin: <https://github.com/green-code-initiative/creedengo-common/blob/main/doc/HOWTO.md#howto-build-the-sonarqube-creedengo-plugins>
+- 2nd step - launch local Sonarqube (with installation of previous local plugin built) : <https://github.com/green-code-initiative/creedengo-common/blob/main/doc/HOWTO.md#howto-install-sonarqube-dev-environment>
+- 3rd step - check that local environment is running perfectly : choose one of repositories with suffix "test-project" (ex : <https://github.com/green-code-initiative/creedengo-java-test-project/tree/main>)
```

Comment:
please use another repo because this one doesn't exist yet ... use "creedengo-python-test-project" and check that this url is OK